sendmail.py

```diff
@@ -52,7 +52,7 @@
 class MailConfig:
     def __init__(self):
         with open("config.yaml", "r") as config_file:
-            for data in yaml.load_all(config_file):
+            for data in yaml.safe_load_all(config_file):
                 if "mail" in data:
```

Comment:
does this actually work? `safe_load_all` should return a list, so the `"mail" in data` should be `False`, no?

Reply:
i'd use plain `safe_load()`.

Reply:
oh, sorry, it's a `for` loop... sure.

Reply:
but yeah, you can lose the `for` loop with `safe_load()`.